openbas-front/src/admin/components/atomic_testings/atomic_testing/TargetResultsDetail.tsx

```diff
@@ -760,7 +753,7 @@ const TargetResultsDetailFlow: FunctionComponent<Props> = ({
                         {emptyFilled(getLabelOfValidationType(injectExpectation))}
                       </div>
                     </div>
-                    {injectExpectation.inject_expectation_type !== 'MANUAL' && (
+                    {!['MANUAL', 'CHALLENGE', 'ARTICLE'].includes(injectExpectation.inject_expectation_type) && (
```

Comment:
Save this list ['MANUAL', 'CHALLENGE', 'ARTICLE'] as a constant somewhere? As it also exists on the backend, we should make sure that if we update it either the backend or the frontend, it should be reflected automatically in the other? Maybe creating an enum in the backend file added to the api-types.d ?

Reply:
Good point, I will changed it